src/io.js

```diff
@@ -146,3 +146,128 @@ export default function twoline2satrec(longstr1, longstr2) {
 
   return satrec;
 }
+
+/* -----------------------------------------------------------------------------
+ *
+ *                           function json2satrec
+ *
+ *  this function converts the OMM json data to variables and initializes the sgp4 
+ *    variables. several intermediate varaibles and quantities are determined. note 
+ *    that the result is a structure so multiple satellites can be processed 
+ *    simultaneously without having to reinitialize. the verification mode is an 
+ *    important option that permits quick checks of any changes to the underlying 
+ *    technical theory. this option works using a modified tle file in which the 
+ *    start, stop, and delta time values are included at the end of the second line
+ *    of data. this only works with the verification mode. the catalog mode simply 
+ *    propagates from -1440 to 1440 min from epoch and is useful when performing 
+ *    entire catalog runs.
+ *
+ *  author        : david vallado                  719-573-2600    1 mar 2001
+ *
+ *  inputs        :
+ *    jsonobj     - OMM json data
+ *    typerun     - type of run                    verification 'v', catalog 'c',
+ *                                                 manual 'm'
+ *    typeinput   - type of manual input           mfe 'm', epoch 'e', dayofyr 'd'
```

Comment:
Only jsonobj variable is still used. You should remove reference to comments from the old function that no longer apply.